homeassistant/components/smhi/weather.py

```diff
@@ -136,7 +136,7 @@ def __init__(
         """Initialize the SMHI weather entity."""
 
         self._attr_name = name
-        self._attr_unique_id = f"smhi-{latitude}-{longitude}"
+        self._attr_unique_id = f"{latitude}, {longitude}"
```

Comment:
The old unique IDs seemed to have been `{lat}-{long}`, not `{lat}, {long}`. Is this an error?

![CleanShot 2022-06-29 at 16 55 52](https://user-images.githubusercontent.com/195327/176468530-549b265d-be44-4832-abda-af82255af54a.png)

Reply:
No, it was with a , for entities while with a - for config entry.

Reply:
Ahh my bad, thanks! 👍 (better to be save than sorry 😬 )